skywalking/__init__.py

```diff
@@ -40,6 +40,7 @@ class Component(Enum):
     Sanic = 7007
     AioHttp = 7008
     Pyramid = 7009
+    Psycopg = 7010
```

Comment:
Remember to add this in main repo and UI repo

Reply:
I've got that done just was waiting for this part to finalize.